application.yml

```diff
@@ -15,79 +15,6 @@
 server:
   port: 8081
```

Comment:
Ideally this whole application.yml would be removed, and setting the server port would be via runtime-configuration environment variables in IntelliJ like:
![image](https://user-images.githubusercontent.com/617828/65288406-a025f780-dafb-11e9-8827-698ec63b4181.png)